### PR TITLE
Prepare broker 20.10.6 / engine 20.10.6 / clib 20.10.3 / connectors 20.10.2 release notes

### PR DESCRIPTION
**Pull request by omercier**
## Description

Release notes **for all poller components**

## Target serie

- [ ] 20.04.x
- [x] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x (master)

### CODE REVIEW

File: fr/releases/centreon-core.md [hunk 1]
```diff
@@ -697,8 +721,43 @@ with the –pool\_size X argument or -s X.
...
+
+### Bug fixes
+
+- Libraries are loaded lazily now. This allows not to check all link issues during the load.
+
+## 20.10.2
+
+### Enhancement
+
+- Compilation in C++14 with conan-center: bintray has stopped. We had to switch to the conan-center. And then our conan dependencies had to upgrade and then we had to switch to C++14. So here is the corresponding compilation.
```
Comment (cg-tw) [inline fr/releases/centreon-core.md:736]:
```suggestion
- Compilation in C++14 with conan-center: bintray has stopped. We had to switch to conan-center. And then our conan dependencies had to be upgraded and then we had to switch to C++14. So here is the corresponding compilation.
```

File: fr/releases/centreon-core.md [hunk 2]
```diff
@@ -697,8 +721,43 @@ with the –pool\_size X argument or -s X.
...
+
+## 20.10.1
+
+`February x, 2021`
+
+### Bugfixes
+
+- Start/Stop process\_manager mechanism is rewritten and much more simpler. The consequences are that there are less dead locks in the processes management.
```
Comment (cg-tw) [inline fr/releases/centreon-core.md:744]:
```suggestion
- Start/Stop process\_manager mechanism is rewritten and much simpler. The consequences are that there are less deadlocks in the processes management.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by chgautier — CHANGES_REQUESTED** [review]

File: en/releases/centreon-core.md [hunk 3]
```diff
@@ -491,6 +499,22 @@ state was HARD even if no notification is configured nor sent.
...
+#### Bugfixes
+
+- A deadlock occasionally appears when broker is stopped right after it started
+- A core dump occasionally appears when Engine is stopped
+- Sometimes Engine cannot reconnect to the central broker when cbd is restarted
+- When many pollers attempt to connect to the central, the last ones may fail to connect
+- Timeranges of timeperiods can't be parsed if they end with `\r` or `\n`
+- Compilation issues on RaspberryPi
+- TLS query not understood by GNUTLS on Redhat 8
+
```
[the remark this answers is not in the document]
Comment (omercier) [reply en/releases/centreon-core.md:513]: It is a duplicate of: `Sometimes Engine cannot reconnect to the central broker when cbd is restarted`

File: fr/releases/centreon-core.md [hunk 4]
```diff
@@ -491,6 +499,22 @@ state was HARD even if no notification is configured nor sent.
...
+#### Correctifs
+
+- Un *deadlock* pouvait se produire lorsque broker était arrêté immédiatement après son démarrage
+- Un *core dump* pouvait apparaître lorsque centengine était stoppé
+- Parfois centengine ne parvenait pas à se reconnecter au broker central
+- Quand beaucoup de collecteurs essayent de se connecter au même moment au broker central, il arrive que les derniers ne puissent pas se connecter
+- Les plages de temps n'étaient pas parsées correctement lorsqu'elles se terminaient par `\r` ou `\n`
+- Problèmes de compilation sur RaspberryPi
+- Les requêtes TLS n'étaient pas correctement traitées par GNUTLS sur RedHat 8
+
```
[the remark this answers is not in the document]
Comment (omercier) [reply fr/releases/centreon-core.md:513]: Same as in English version: it duplicates `Sometimes Engine cannot reconnect to the central broker when cbd is restarted
`

File: en/releases/centreon-core.md [hunk 5]
```diff
@@ -372,6 +372,14 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Engine
 
+### 20.10.6
+
```
Comment (chgautier) [inline en/releases/centreon-core.md:376]:
```suggestion

`July 15, 2021`

```

File: en/releases/centreon-core.md [hunk 6]
```diff
@@ -491,6 +499,22 @@ state was HARD even if no notification is configured nor sent.
 
 ## Centreon Broker
 
+### 20.10.6
+
```
Comment (chgautier) [inline en/releases/centreon-core.md:503]:
```suggestion

`July 15, 2021`

```

File: en/releases/centreon-core.md [hunk 7]
```diff
@@ -693,8 +717,45 @@ with the –pool\_size X argument or -s X.
 
 - Contains all fixes up to version 20.04.9
 
+## Centreon CLib
+
+## 20.10.3
+
```
Comment (chgautier) [inline en/releases/centreon-core.md:723]:
```suggestion

`July 15, 2021`

```

File: en/releases/centreon-core.md [hunk 8]
```diff
@@ -693,8 +717,45 @@ with the –pool\_size X argument or -s X.
...
+`October 21, 2020`
+
+### Bugfixes
+
+- A possible deadlock was removed when a process is added and at the same time a process is removed because in timeout.
+
 ## Centreon Connector Perl
 
+### 20.10.2
+
```
Comment (chgautier) [inline en/releases/centreon-core.md:756]:
```suggestion

`July 15, 2021`

```

File: en/releases/centreon-core.md [hunk 9]
```diff
@@ -707,6 +768,10 @@ with the –pool\_size X argument or -s X.
 
 ## Centreon Connector SSH
 
+### 20.10.2
+
```
Comment (chgautier) [inline en/releases/centreon-core.md:772]:
```suggestion

`July 15, 2021`

```

File: fr/releases/centreon-core.md [hunk 10]
```diff
@@ -372,6 +372,14 @@ commerciales, vous pouvez vous rendre sur notre
 
 ## Centreon Engine
 
+### 20.10.6
+
```
Comment (chgautier) [inline fr/releases/centreon-core.md:376]:
```suggestion

`15 juillet 2021`

```

File: fr/releases/centreon-core.md [hunk 11]
```diff
@@ -491,6 +499,22 @@ state was HARD even if no notification is configured nor sent.
 
 ## Centreon Broker
 
+### 20.10.6
+
```
Comment (chgautier) [inline fr/releases/centreon-core.md:503]:
```suggestion

`15 juillet 2021`

```

File: fr/releases/centreon-core.md [hunk 12]
```diff
@@ -697,8 +721,45 @@ with the –pool\_size X argument or -s X.
 
 - Contient tous les correctifs jusqu'à la version 20.04.9
 
+## Centreon CLib
+
+## 20.10.3
+
```
Comment (chgautier) [inline fr/releases/centreon-core.md:727]:
```suggestion

`15 juillet 2021`

```

File: fr/releases/centreon-core.md [hunk 13]
```diff
@@ -697,8 +721,45 @@ with the –pool\_size X argument or -s X.
...
+`October 21, 2020`
+
+### Bugfixes
+
+- A possible deadlock was removed when a process is added and at the same time a process is removed because in timeout.
+
 ## Centreon Connector Perl
 
+### 20.10.2
+
```
Comment (chgautier) [inline fr/releases/centreon-core.md:760]:
```suggestion

`15 juillet 2021`

```

File: fr/releases/centreon-core.md [hunk 14]
```diff
@@ -711,6 +772,10 @@ with the –pool\_size X argument or -s X.
 
 ## Centreon Connector SSH
 
+### 20.10.2
+
```
Comment (chgautier) [inline fr/releases/centreon-core.md:776]:
```suggestion

`15 juillet 2021`

```

**Review by chgautier — CHANGES_REQUESTED** [review]

**Review by chgautier — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 15]
```diff
@@ -707,6 +776,14 @@ with the –pool\_size X argument or -s X.
...
+`July 15, 2021`
+
+`July 15, 2021`
```
Comment (chgautier) [inline en/releases/centreon-core.md:783]:
```suggestion
```

File: en/releases/centreon-core.md [hunk 16]
```diff
@@ -707,6 +776,14 @@ with the –pool\_size X argument or -s X.
 
 ## Centreon Connector SSH
 
+### 20.10.2
+
+`July 15, 2021`
+
```
Comment (chgautier) [inline en/releases/centreon-core.md:782]:
```suggestion
```

**Review by chgautier — CHANGES_REQUESTED** [review]

File: en/releases/centreon-core.md [hunk 17]
```diff
@@ -693,8 +721,49 @@ with the –pool\_size X argument or -s X.
 
 - Contains all fixes up to version 20.04.9
 
+## Centreon CLib
+
+## 20.10.3
```
Comment (chgautier) [inline en/releases/centreon-core.md:726]:
```suggestion
### 20.10.3
```

File: en/releases/centreon-core.md [hunk 18]
```diff
@@ -693,8 +721,49 @@ with the –pool\_size X argument or -s X.
...
+
+`July 15, 2021`
+
+### Bug fixes
```
Comment (chgautier) [inline en/releases/centreon-core.md:730]:
```suggestion
#### Bug fixes
```

File: en/releases/centreon-core.md [hunk 19]
```diff
@@ -693,8 +721,49 @@ with the –pool\_size X argument or -s X.
...
+
+- Libraries are loaded lazily now. This allows not having to check all link issues during loading.
+
+## 20.10.2
```
Comment (chgautier) [inline en/releases/centreon-core.md:734]:
```suggestion
### 20.10.2
```

File: en/releases/centreon-core.md [hunk 20]
```diff
@@ -693,8 +721,49 @@ with the –pool\_size X argument or -s X.
...
+
+`June 4, 2021`
+
+### Enhancement
```
Comment (chgautier) [inline en/releases/centreon-core.md:738]:
```suggestion
#### Enhancements
```

File: en/releases/centreon-core.md [hunk 21]
```diff
@@ -693,8 +721,49 @@ with the –pool\_size X argument or -s X.
...
+
+- Compilation in C++14 with conan-center: bintray has stopped. We had to switch to conan-center. And then our conan dependencies had to be upgraded and then we had to switch to C++14. So here is the corresponding compilation.
+
+## 20.10.1
```
Comment (chgautier) [inline en/releases/centreon-core.md:742]:
```suggestion
### 20.10.1
```

File: en/releases/centreon-core.md [hunk 22]
```diff
@@ -693,8 +721,49 @@ with the –pool\_size X argument or -s X.
...
+
+`April 28, 2021`
+
+### Bugfixes
```
Comment (chgautier) [inline en/releases/centreon-core.md:746]:
```suggestion
#### Bugfixes
```

File: en/releases/centreon-core.md [hunk 23]
```diff
@@ -693,8 +721,49 @@ with the –pool\_size X argument or -s X.
...
+
+- Start/Stop process\_manager mechanism is rewritten and much simpler. The consequences are that there are less deadlocks in the processes management.
+- The simple cases that are addition, subtraction and some other cases, have had their computations simplified, that is to say less computations, so faster.
+
+## 20.10.0
```
Comment (chgautier) [inline en/releases/centreon-core.md:751]:
```suggestion
### 20.10.0
```

File: en/releases/centreon-core.md [hunk 24]
```diff
@@ -693,8 +721,49 @@ with the –pool\_size X argument or -s X.
...
+
+`October 21, 2020`
+
+### Bugfixes
```
Comment (chgautier) [inline en/releases/centreon-core.md:755]:
```suggestion
#### Bugfixes
```

File: fr/releases/centreon-core.md [hunk 25]
```diff
@@ -697,8 +725,49 @@ with the –pool\_size X argument or -s X.
 
 - Contient tous les correctifs jusqu'à la version 20.04.9
 
+## Centreon CLib
+
+## 20.10.3
```
Comment (chgautier) [inline fr/releases/centreon-core.md:730]:
```suggestion
### 20.10.3
```

File: fr/releases/centreon-core.md [hunk 26]
```diff
@@ -697,8 +725,49 @@ with the –pool\_size X argument or -s X.
...
+
+`15 juillet 2021`
+
+### Bug fixes
```
Comment (chgautier) [inline fr/releases/centreon-core.md:734]:
```suggestion
#### Bug fixes
```

File: fr/releases/centreon-core.md [hunk 27]
```diff
@@ -697,8 +725,49 @@ with the –pool\_size X argument or -s X.
...
+
+- Libraries are loaded lazily now. This allows not having to check all link issues during loading.
+
+## 20.10.2
```
Comment (chgautier) [inline fr/releases/centreon-core.md:738]:
```suggestion
### 20.10.2
```

File: fr/releases/centreon-core.md [hunk 28]
```diff
@@ -697,8 +725,49 @@ with the –pool\_size X argument or -s X.
...
+
+`4 juin 2021`
+
+### Enhancement
```
Comment (chgautier) [inline fr/releases/centreon-core.md:742]:
```suggestion
#### Enhancements
```

File: fr/releases/centreon-core.md [hunk 29]
```diff
@@ -697,8 +725,49 @@ with the –pool\_size X argument or -s X.
...
+
+- Compilation in C++14 with conan-center: bintray has stopped. We had to switch to conan-center. And then our conan dependencies had to be upgraded and then we had to switch to C++14. So here is the corresponding compilation.
+
+## 20.10.1
```
Comment (chgautier) [inline fr/releases/centreon-core.md:746]:
```suggestion
### 20.10.1
```

File: fr/releases/centreon-core.md [hunk 30]
```diff
@@ -697,8 +725,49 @@ with the –pool\_size X argument or -s X.
...
+
+`28 avril 2021`
+
+### Bugfixes
```
Comment (chgautier) [inline fr/releases/centreon-core.md:750]:
```suggestion
#### Bugfixes
```

File: fr/releases/centreon-core.md [hunk 31]
```diff
@@ -697,8 +725,49 @@ with the –pool\_size X argument or -s X.
...
+
+- Start/Stop process\_manager mechanism is rewritten and much simpler. The consequences are that there are less deadlocks in the processes management.
+- The simple cases that are addition, subtraction and some other cases, have had their computations simplified, that is to say less computations, so faster.
+
+## 20.10.0
```
Comment (chgautier) [inline fr/releases/centreon-core.md:755]:
```suggestion
### 20.10.0
```

File: fr/releases/centreon-core.md [hunk 32]
```diff
@@ -697,8 +725,49 @@ with the –pool\_size X argument or -s X.
...
+
+`October 21, 2020`
+
+### Bugfixes
```
Comment (chgautier) [inline fr/releases/centreon-core.md:759]:
```suggestion
#### Bugfixes
```

**Review by chgautier — CHANGES_REQUESTED** [review]

**Review by chgautier — APPROVED** [review]

**Pull request merged**